### DISCUSSION TITLE
Tips and questions

### DISCUSSION THREAD

**Discussion by gacarita** [giordano/LombScargle.jl#33] [Q&A]
Hello there,

My name is Gabriel i wanna ask something here.

I'm using lombscargle periodogram to verify some frequencys in my orbits dynamics. I'm starting with a simple example a mean motion resonance 1:1.

![image](https://user-images.githubusercontent.com/53542579/173152528-6175c2e2-36db-4d99-8113-59da18603fa6.png)

![image](https://user-images.githubusercontent.com/53542579/173152246-5f1cef64-0e12-4671-bc1a-df0e29ac6ea1.png)

In this case i'm getting what i want (1 period of peak). However if i follow the best fit frequency model described in the docs i have the result below.

![image](https://user-images.githubusercontent.com/53542579/173152358-92718379-9f4e-4fd7-b7b3-75a2b15ec2dd.png)

I'm not sure if the fitting model is working very well for my model, there is something i could be doing wrong ? I would love some tips to help me understand this tool.

**Comment by giordano** [comment]
As [explained in the documentation](https://juliaastro.github.io/LombScargle.jl/stable/#LombScargle.model-Function) and on [Wikipedia](https://en.wikipedia.org/wiki/Least-squares_spectral_analysis#The_generalized_Lomb%E2%80%93Scargle_periodogram), the Lomb-Scargle periodogram uses functions of the type $a_{f} \cos(\omega t) + b_{f}\sin(\omega t) + c_{f}$.  With a single frequency $f$ it's highly unlikely you'll be able to fit your data, unless it's purely a sum of a sine and a cosine of the same frequency.  You can get better results by finding the next frequencies.  The documentation has an [example of using models with multiple frequencies](https://juliaastro.github.io/LombScargle.jl/stable/#Find-the-Best-Fitting-Model), but in general Lomb-Scargle isn't for closely fitting completely arbitrary data, the main use case is for finding the main period.
